configs/build.tslint.json

```diff
@@ -1,7 +1,8 @@
-// Lint rules to use when building.  We don't include warnings to avoid
-// polluting the build output.
 {
+  "//": [
+    "Lint rules to use when building. We don't include warnings to avoid polluting the build output."
+  ],
   "extends": [
     "./errors.tslint.json"
   ]
-}
+}
```

Comment:
Could you configure your editor to keep the new line at the end of files?

Reply:
I thought this is a project specific configuration, how can I set it in VS Code?

Reply:
We can set the `"files.insertFinalNewline": true` editorconfig property it seems.  I will try it and open a PR about it.

https://github.com/Microsoft/vscode/issues/12076

Reply:
Actually, we already have this in .editorconfig:

    insert_final_newline = true

Do you have the editorconfig extension for vscode?

Reply:
No, I am not using extensions.

Reply:
Ok, please see

https://github.com/theia-ide/theia/pull/3123